tests/test_xtalimaging.py

```diff
@@ -16,7 +18,8 @@ def test_xtal_imaging(testdb):
     assert sid is not None
     assert sid > 0
 
-    si_full_path = "/dls/i03/data/2018/cm14451-99/something_new.jpg"
+    secs = int(datetime.datetime.now().timestamp())
+    si_full_path = "/dls/i03/data/2018/cm14451-99/something_new_%d.jpg" % secs
```

Comment:
Possibly clearer/more modern Python to use f-String here?
```suggestion
    si_full_path = f"/dls/i03/data/2018/cm14451-99/something_new_{secs}.jpg"
```

Reply:
Good point, done!